scanapp/views.py

```diff
@@ -31,21 +32,24 @@
 from django.http import HttpResponse
 from django.http import HttpResponseRedirect
 from django.shortcuts import render
+
 from django.utils import timezone
 from django.views import View
 from django.views.decorators.csrf import csrf_exempt
 from django.views.generic.base import TemplateView
 from django.views.generic.edit import FormView
+from giturl import *
+from rest_framework.authtoken.models import Token
 
 from rest_framework.authtoken.models import Token
 
 from scanapp.forms import LocalScanForm
 from scanapp.forms import UrlScanForm
-
 from scanapp.models import Scan
 
 from scanapp.tasks import apply_scan_async
 from scanapp.tasks import create_scan_id
+from scanapp.tasks import handle_special_urls
 from scanapp.tasks import scan_code_async
 
 from django.views.decorators.csrf import csrf_exempt
```

Comment:
Django imports should go with the thirdparty, before you scanapp import (e.g. 4 blocks: 1. `__*` imports, 2. stdlib imports, 3. thirdparty imports, 4. own code imports)